lib/bootsy/container.rb

```diff
@@ -11,43 +11,41 @@ module Container
     extend ActiveSupport::Concern
 
     included do
-      class_eval do
-        has_one :bootsy_image_gallery,
-                class_name: 'Bootsy::ImageGallery',
-                as: :bootsy_resource,
-                dependent: :destroy
+      has_one :bootsy_image_gallery,
+              class_name: 'Bootsy::ImageGallery',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.